h2o-core/src/main/java/water/api/schemas3/TwoDimTableV3.java

```diff
@@ -172,19 +172,19 @@ public TwoDimTable fillImpl(TwoDimTable impl) {
     for (int r=0; r<data[0].length; ++r) {
       for (int c=0; c<data.length; ++c) {
         try {
-          if (columns[c].format == "string") {
+          if (columns[c].format.equals("string")) {  // switch(String) is not java1.6 compliant!
```

Comment:
Minor comment: `switch(String)` will not work with javac 1.6 but, at the byte code level, it should be ok

btw: good catch!